lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -334,7 +340,7 @@ def install_puppet_agent_on(hosts, opts = {})
           opts[:puppet_collection] ||= 'pc1' #hi!  i'm case sensitive!  be careful!
           opts[:puppet_agent_version] ||= opts[:version] #backwards compatability with old parameter name
 
-          block_on hosts do |host|
+          block_on hosts, { :run_in_parallel => @options[:run_in_parallel].include('install') } do |host|
```

Comment:
should this not be passable from the `opts` hash as well?

should this go into a little helper method so that it's done the same way in all of these methods?